javascript/src/test/components/TutorAssignment/TutorAssignmentSelect.test.js

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import { render, waitFor } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import TutorAssignmentSelect from "main/components/TutorAssignment/TutorAssignmentSelect";
+import { selectorIndices } from "main/components/TutorAssignment/TutorAssignmentSelect";
+
+import { quarterProvider, courseProvider, tutorAssignmentProvider } from "main/services/selectorSupport"
+jest.mock("main/services/selectorSupport");
+
+
+describe("TutorAssignmentSelect", () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it("renders a usable tutor assignment UI", async () => {
+    const onChange = jest.fn();
+    quarterProvider.mockResolvedValue(["20201"]);
+    courseProvider.mockResolvedValue([{ id: 1, name: "CMPSC 156" }]);
+    tutorAssignmentProvider.mockResolvedValue([{ id: 2, tutor: { firstName: "Foo", lastName: "Bar" }}]);
+    const { _debug, getByPlaceholderText, getByText } = render(
```

Comment:
For consistency with other tests, could we remove the unused variable here?
```suggestion
    const { getByPlaceholderText, getByText } = render(
```